### ISSUE TITLE
CLI should emit Jupyter notebook token (and/or allow password to be specified)

### ISSUE THREAD

**Issue by BrandonHaynes** [uwescience/myria-ec2-ansible#38]
The 4.3 release of Jupyter enables token-based authentication by default, and so a user will not be able to access notebooks in a newly-launched cluster without SSHing to retrieve this token.  (This token also changes every time the notebook service is recycled.)

It would be useful to at least emit this value to stdout for a newly-launched cluster (via `sudo -u myria jupyter notebook list`).  Better would be to set a static password by adding the following line to `/etc/jupyter/jupyter_notebook_config.py`:

```conf
c.NotebookApp.password = 'sha:XXXXX'
```

The hash can be obtained using the following script:

```python
from IPython.lib import passwd
passwd(passphrase='12345')
```

**Comment by senderista** [comment]
Per discussion, we'll disable auth by default, but add a CLI switch for specifying a password.

**Comment by senderista** [comment]
fixed in cd6d638
